server/settings/publish_plugins.py

```diff
@@ -221,7 +221,12 @@ class OIIOToolArgumentsModel(BaseSettingsModel):
 
 class ExtractOIIOTranscodeOutputModel(BaseSettingsModel):
     _layout = "expanded"
-    name: str = SettingsField("", title="Name")
+    name: str = SettingsField(
+        "",
+        title="Name",
+        description="Output name (no space)",
```

Comment:
Actually i was thinking about something like that. Not just for regexes - pydantic gives a pretty good (but technical) responses. It would be awesome to have "explain" button in the error message.

Reply:
Issue why this enhancement had been created at first place is that we are currently allowing users to save the names with spaces and users might not read the description. Then it will be broken downstream and it is very difficult to debug. I would rather keep the field regex argument as part of this PR's solution. Only reason why I was pointing at you was so you are aware of the issue of the very technical message. This should not be blocking this PR from merge.

Reply:
I don't think they are saying to drop the regex validation - they are saying to elaborate more in the `description` argument what the actual valid characters are, right @Innders ?

Reply:
@BigRoy is correct, I don't care about the regex validation personally but the description could be better.

Can't be mad if you ask for my review and I give it ;)

Reply:
> Can't be mad if you ask for my review and I give it ;)

haha. yeah, this is one of those **lost in translation** cases. It has appeared to me you want me to remove the regex for now since you were requesting changes.

Your proposal for `regex_description` was the right one.